auth-server/test/resolvers/mutation/addressUnlink.spec.ts

```diff
@@ -0,0 +1,75 @@
+import 'mocha';
+import { expect } from 'chai';
+import { AuthenticationError } from 'apollo-server';
+import { uuid } from 'uuidv4';
+
+import User from '../../../src/model/User';
+import Address from '../../../src/model/Address';
+import addressUnlink from '../../../src/resolvers/mutation/addressUnlink';
+import signup from '../../../src/resolvers/mutation/signup';
+import { Context } from '../../../src/types';
+import messages from '../../../src/utils/messages';
+
+describe('addressUnlink mutation', () => {
+	let signupResult;
+	let dbAddress;
+	const fakectx: Context = {
+		req: {
+			headers: {}
+		},
+		res: {
+			cookie: () => {}
+		}
+	} as any;
+	const email = 'test@email.com';
+	const password = 'testpass';
+	const username = 'testuser';
+	const name = 'test name';
+
+	before(async () => {
+		signupResult = await signup(null, { email, password, username, name }, fakectx);
+		fakectx.req.headers.authorization = `Bearer ${signupResult.token}` // eslint-disable-line
+
+		dbAddress = await Address
+			.query()
+			.allowInsert('[network, address, user_id, public_key, sign_message, linked]')
+			.insert({
+				network: 'kasuma',
+				address: uuid(),
```

Comment:
Let's use any address here instead of `uuid`

Reply:
done